modules/back-up-quay-procedure.adoc

```diff
@@ -0,0 +1,191 @@
+example-registry-clair-app-7dccddc6fd-fwckp            1/1     Running     0          69m
+example-registry-clair-postgres-558d7cc9c-xgk2m        1/1     Running     0          68m
+example-registry-quay-app-5677cb59b7-fpxcg             1/1     Running     0          69m
+example-registry-quay-app-upgrade-zstr4                0/1     Completed   0          69m
+example-registry-quay-config-editor-67c769f669-dp5gf   1/1     Running     0          69m
+example-registry-quay-database-6d58c4c545-99s2g        1/1     Running     0          69m
+example-registry-quay-mirror-d4bc9c99c-rmk4f           1/1     Running     0          68m
+example-registry-quay-postgres-init-rwbsq              0/1     Completed   0          69m
+example-registry-quay-redis-577776d494-k48zv           1/1     Running     0          69m
+quay-operator.{productminv}-798d5c8b69-q6x88                  1/1     Running     0          79m
+----
+
+. Obtain the configuration secret of your Quay application deployment in JSON:
++
+----
+$ oc get deployment -n quay-enterprise example-registry-quay-app -o json | jq '.spec.template.spec.containers[].env[]'
+----
++
+Example output:
++
+[source,terminal]
+----
+{
+  "name": "QE_K8S_CONFIG_SECRET",
+  "value": "example-registry-quay-config-secret-m25mf4dg78"
+}
+{
+  "name": "QE_K8S_NAMESPACE",
+  "valueFrom": {
+    "fieldRef": {
+      "apiVersion": "v1",
+      "fieldPath": "metadata.namespace"
+    }
+  }
+}
+{
+  "name": "DEBUGLOG",
+  "value": "false"
+}
+{
+  "name": "WORKER_COUNT_WEB",
+  "value": "4"
+}
+{
+  "name": "WORKER_COUNT_SECSCAN",
+  "value": "2"
+}
+{
+  "name": "WORKER_COUNT_REGISTRY",
+  "value": "8"
+}
+----
+
+. Back up the Quay configuration secret into a yaml file, for example `quay-config-yaml.backup.yaml`:
++
+----
+$ oc get secret -n quay-enterprise example-registry-quay-config-secret-m25mf4dg78 -o json | jq '.data."config.yaml"' | cut -d '"' -f2 | base64 -d -w0 > quay-config-yaml-backup.yaml
+----
+
+. Obtain the DB_URI of your Quay database from the generated yaml file:
++
+----
+$ cat quay-config-yaml-backup.yaml | grep -i DB_URI | cut -d ':' -f3 | sed 's/^..//g'
+----
++
+Example output:
++
+----
+example-registry-quay-database
+----
+
+. Back up your Quay database locally using PostgreSQL:
++
+----
+$ oc exec -it -n quay-enterprise example-registry-quay-database-7885795468-x9v6n -- pg_dump -h localhost -d example-registry-quay-database -O > quay-database-backup.sql
+----
+
+. Ensure that your database has been backed up by running the following command:
++
+----
+$ ls -lah
+----
++
+Example output:
++
+----
+total 248K
+drwxr-xr-x  2 root root   74 Jan 28 15:00 .
+drwx------ 12 root root  298 Jan 28 14:42 ..
+-rw-r--r--  1 root root 1.9K Jan 28 14:56 quay-config-yaml-backup.yaml
+-rw-r--r--  1 root root 243K Jan 28 15:00 quay-database-backup.sql
+----
+
+. Obtain the OpenShift Container Storage credentials of your Quay configuration using the backup yaml file:
```

Comment:
This likely needs to be its own section. It also needs to be better explained. 

In short what this is doing is backing up the OCS storage to AWS (that's not clear as you start reading this section), I also don't think many customers are going to do this. If they are/were they would have likely chosen AWS as their storage vendor over OCS (in the first place). 

While its fine to have instructions like this we need to have something that goes from OCS to local disk (as a backup). 

@mhackett do we have OCS docs we could link to that tell user how to backup buckets to local storage?